MapboxVision/VisionManager/VisionReplayManager.swift

```diff
@@ -76,7 +77,7 @@ public final class VisionReplayManager: BaseVisionManager {
     }
 
     /**
-     Cleanup the state and resources of `VisionManger`.
+     Cleanup the state and resources of `VisionReplayManager`.
```

Comment:
Should we add info about when to should/should not call `destroy` method?

Reply:
Please propose in https://github.com/mapbox/mapbox-vision/pull/1042/. This is just a mirror.
Considering particular thing such description is added to create method.